spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ReviewsController, type: :controller do
+  before(:each) do
+    fake_reviews = []
+    11.times do
+      fake_reviews.push(FactoryGirl.create(:review))
+    end
+  end
+
+  describe "GET#index" do
+    it "should return a list of the 10 latest reviews (most recent first)" do
+      get :index
+      returned_json = JSON.parse(response.body)
+      reviews = returned_json["reviews"]
+
+      expect(reviews.length).to eq(10)
+      9.times do |idx|
+        expect(reviews[idx]["updated_at"] > reviews[idx + 1]["updated_at"])
+      end
+    end
+  end
+end
```

Comment:
Ill allow this, but with a suggestion

It is generally better to avoid looping with the creation of your test DB objects and your expect statements 
This generally does the trick, but you technically really only need 2 reviews to be created to really test this, which you could easily do with just 1 or 2 lines of code and a `let!()` statements. 

Actually...what are these reviews for? Did you not need to create movie id, do you just have reviews index page not linked up to any specific movie?